packages/p2p/src/components/online-status/__tests__/online-status-label.spec.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import OnlineStatusLabel from '../online-status-label';
+import moment from 'moment';
+import 'moment/min/locales';
+import { isMobile, toMoment } from '@deriv/shared';
+
+const props = {
+    is_online: 0,
+    last_online_time: 0,
+};
+
+let mock_value: moment.Moment = moment();
+jest.mock('@deriv/shared', () => ({
+    ...jest.requireActual('@deriv/shared'),
+    toMoment: jest.fn(() => mock_value),
+    isMobile: jest.fn(() => false),
+}));
+
+describe('<OnlineStatusLabel/>', () => {
+    it('should render the component with default state', () => {
+        render(<OnlineStatusLabel {...props} />);
+
+        expect(screen.getByText('Seen more than 6 months ago')).toBeInTheDocument();
+    });
+    it('should render component with default state in responsive view', () => {
+        (isMobile as jest.Mock).mockReturnValue(true);
+        render(<OnlineStatusLabel {...props} />);
+
+        expect(screen.getByText('Seen more than 6 months ago')).toBeInTheDocument();
+    });
+    it('should render component when last seen is not given', () => {
+        const newprops = { ...props, is_online: 1 };
+        render(<OnlineStatusLabel {...newprops} />);
+
+        expect(screen.getByText('Online')).toBeInTheDocument();
+    });
+    it('should render component when last seen is in hours', () => {
```

Comment:
```suggestion
    it('should show "Seen 3 hours ago" label when "last_online_time" property is in hours', () => {
```

Reply:
should we be giving the exact statement and property value name? and since last_online_time property is not being passed in hours i dont think we can give it this way.. shall i give it this way instead? 
`it('should show label "Seen 3 hours ago" when last seen is in hours', () => {`